### PR TITLE
ardour: update to 6.3.

### PR DESCRIPTION
**Pull request by sgn**
Close: #24930

@q66  Please see if I screwed up powerpc, I don't know what is going on with armv[67] build, it try to simulate neon? I don't know.

### CODE REVIEW

File: srcpkgs/ardour/template [hunk 1]
```diff
@@ -1,22 +1,47 @@
 # Template file for 'ardour'
 pkgname=ardour
-version=6.0
+version=6.3
 revision=1
+wrksrc="Ardour-${version}.0"
```
Comment (Chocimier) [inline srcpkgs/ardour/template:5]: Why not add zero to version?

File: srcpkgs/ardour/template [hunk 2]
```diff
@@ -1,22 +1,47 @@
...
 build_style=waf3
 configure_args="--cxx11 --no-phone-home --with-backends=jack,alsa,dummy
- --libjack=weak --optimize --docs --use-external-libs"
-hostmakedepends="python3 git graphviz doxygen pkg-config clang perl"
+ --libjack=weak --optimize --docs --use-external-libs --freedesktop"
+hostmakedepends="python3 git graphviz doxygen pkg-config clang perl
```
Comment (Chocimier) [inline srcpkgs/ardour/template:9]: Is git still needed?

File: srcpkgs/ardour/template [hunk 3]
```diff
@@ -1,22 +1,47 @@
...
 
-do_fetch() {
-	git clone --depth 1 --branch=$version https://github.com/Ardour/ardour $pkgname-$version
+case "$XBPS_TARGET_MACHINE" in
+	x86_64*)  configure_args+=" --dist-target=x86_64" ;;
+	i686*)    configure_args+=" --dist-target=i686" ;;
+	aarch64*) configure_args+=" --dist-target=aarch64" ;;
+	arm*)     broken="glibmm/threads.h not found, wth is going on?" ;;
+	# Some armv7 doesn't have NEON
+	armv7*)   configure_args+=" --dist-target=armhf --no-fpu-optimization" ;;
```
Comment (Chocimier) [inline srcpkgs/ardour/template:29]: Looks unreachable.

File: srcpkgs/ardour/template (hunk 1 above)
Comment (sgn) [reply srcpkgs/ardour/template:5]: I followed there tag schema, will add 0 into it.

File: srcpkgs/ardour/template (hunk 2 above)
Comment (sgn) [reply srcpkgs/ardour/template:9]: No

File: srcpkgs/ardour/template (hunk 3 above)
Comment (sgn) [reply srcpkgs/ardour/template:29]: Yes, intentionally left those configuration until I figure out what's the problem with armv6 and armv7

**Comment by q66** [comment]
you should check if the ARM errors are cross-related

i don't see anything bad for ppc, but i'll test it after it's merged, i just don't have time to deal with everything

**Comment by sgn** [comment]
> i don't see anything bad for ppc, but i'll test it after it's merged, i just don't have time to deal with everything

OK, I see one of your patch there. I'll push with current version first and figure armv[67] later.

**Pull request closed without merging**